src/Shared/Recruit.Shared.Web/Extensions/AddressExtensions.cs

```diff
@@ -0,0 +1,14 @@
+using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+
+namespace Recruit.Shared.Web.Extensions
+{
+    public static class AddressExtensions
+    {
+        public static string ToAddressString(this IAddress address)
```

Comment:
I can see why you've introduced IAddress however the interface on the POCO is tying 2 unrelated address items together unnecessarily (and we don't really want interfaces on POCOs).  Personally I'd implement ToAddressString in each POCO. Maybe they can both call a static Domain helper function E.g. `AddressHelper.GetAddressString(addressLine1, addressLine2, addressLine3, etc....)`.